inspirehep/utils/helpers.py

```diff
@@ -78,3 +78,12 @@ def force_force_list(data):
     elif isinstance(data, (tuple, set)):
         return list(data)
     return data
+
+def is_deleted(data):
+    """Check if data is deleted.
+    """
+
+    if data.lower() == 'deleted':
+        return True
+    else:
+        return False
```

Comment:
One more tiny comment. For consistency: we are calling the argument `value` rather than `data` :dancers: :